mrp_production_request/models/mrp_production_request.py

```diff
@@ -91,9 +91,14 @@ def _compute_manufactured_qty(self):
         track_visibility='onchange',
         readonly=True, states={'draft': [('readonly', False)]})
     description = fields.Text('Description')
-    date_planned = fields.Date(
-        string='Planned Date', required=True, track_visibility='onchange',
-        default=fields.Date.context_today)
+    date_planned_start = fields.Datetime(
+        'Deadline Start', copy=False, default=fields.Datetime.now,
+        index=True, required=True,
+        states={'confirmed': [('readonly', False)]}, oldname="date_planned")
+    date_planned_finished = fields.Datetime(
```

Comment:
This is making me think :thinking: Maybe the wizard should be extended with a date field that is defaulted to this but can be changed. In such case I think that makes more sense to keep the `date_planned` as the only date field in the manufacture request, i.e. the date in which all the quantity must have been made. 

What do you think?

Reply:
I know quite little of this module, but I'm starting to think that the `mrp.production.request` should be as similar as possible to `mrp.production` (also in order to solve the *issue* I told you about [here](https://github.com/OCA/manufacture/pull/288/files/2c743e97f1774a4d994ae71d6fad3f5a4493af65#r203273104)), so I'd leave the fields of `mrp.production.request` as they are int this PR.
Anyway your suggestion is a functionality that we can still add: the only difference is that `date_planned` of the wizard will default to `date_planned_start` of the request, would this be ok with you?

Reply:
That's fine, don't worry about the new functionality I note it as a possible improvement. 

Regarding the renamed field (date_planned -> date_planned_start), could you please add a migration script?

Reply:
Done in https://github.com/OCA/manufacture/pull/288/commits/f1273cba85ae97c0f11e742a964c942e2796a273, I don't know if I did it correctly for the migration, I never did that before, could you please check?